draft-ietf-cats-usecases-requirements.md

```diff
@@ -770,6 +770,134 @@ informative:
    WAN1 -> Cloud2 vCPE1 -> Cloud2 APP1
 
 
+##  Computing-Aware AI large model
+   
+   AI large model refers to a type of artificial intelligence model that 
+   is trained on massive amounts of data using deep learning techniques. 
+   These models are characterized by their large size, high complexity, 
+   and high computational requirements. AI large models have become 
+   increasingly important in various fields, such as natural language 
+   processing, computer vision, and speech recognition.
+   
+   There are usually two types of AI large models, AI foundation model 
```

Comment:
The subtle "usualy" may be confusing for some readers absent an authoritative reference to cite;

Reply:
Agree to delete "usually".